### ISSUE TITLE
The button is selected by two rectangles on IE

### ISSUE THREAD

**Issue by Helen09** [Microsoft/pxt#4135]
**Repro Steps:**
• Navigate to https://makecode.microbit.org/beta
• Click "More" -> "High Contrast On" and then click "Projects" on the top-left
• Switch form "My Stuff" to "Projects" and "Examples" and Observe
**Expected result:(Chrome, Edge & Firefox )**
The button should be selected by one green rectangle
![compare1](https://user-images.githubusercontent.com/23466737/38792955-d065d256-413e-11e8-9f1e-868d1ec9cd89.gif)
**Actual result:(IE)**
The button is selected by two rectangles on IE
![issue1](https://user-images.githubusercontent.com/23466737/38792940-c8b7d838-413e-11e8-9e32-acf37d11e34a.gif)

**Comment by samelhusseini** [comment]
Won't fix.
